qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -1668,6 +1680,18 @@ def _on_select_client_certificate(self, selection):
         else:
             selection.selectNone()
 
+    def _on_recommended_state_changed(
+            self,
+            recommended_state: QWebEnginePage.LifecycleState
+    ) -> None:
+        log.webview.debug("Recommended state changed to: {}".format(recommended_state))
```

Comment:
With the check below dropped, this seems a bit redundant with the call below maybe?